cli/CMakeLists.txt

```diff
@@ -8,5 +8,6 @@ set(QT_LIBRARIES Qt5::Core)
 file(GLOB_RECURSE SOURCES "src/*.cpp")
 include_directories("src/" "../lib/src/" "..")
 
-add_executable(${PROJECT_NAME} ${SOURCES})
-target_link_libraries(${PROJECT_NAME} ${QT_LIBRARIES} lib)
+add_executable("Grabber-cli" ${SOURCES})
```

Comment:
I'm really not a big fan of giving final names before the install step.
Can't this renaming be done with the `RENAME` argument of install() instead?